package.json

```diff
@@ -52,9 +49,12 @@
     "yargs-parser": "^21.0.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.17.0",
     "@babel/eslint-parser": "^7.17.0",
     "@babel/plugin-proposal-class-properties": "^7.0.0",
+    "@babel/plugin-syntax-dynamic-import": "^7.8.0",
```

Comment:
I would actually remove `@babel/plugin-syntax-dynamic-import` entirely from there, removing the explicit registration of that plugin in the Encore-generated config, because this plugin is part of `@babel/preset-env` since years now.

Reply:
Let's go for it, thanks